reactnative/README.md

```diff
@@ -0,0 +1,66 @@
+# React Native Input Mask
+
+### This is beta and is a still a WIP!
```

Comment:
Can you change this line to something like

```
&#x1F6A7; **This component is experimental**. It works but it's not production-ready yet.
```

The reason I prefer "experimental" to "WIP" is because "WIP" indicates: "Don't install, I'm still working on this" while "experimental" says "Feel free to install and play around, but don't use in production". The emoji code is for this 🚧.